extras/build_on_circle.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-./in_parallel.sh "make RM=" $(find . -maxdepth 2 -name *.go -printf "%h\n" | sort -u | sed -n 's/\.\/\(.*\)/\1\/\1/p')
+./in_parallel.sh "make RM=" "$(find . -maxdepth 2 -name "./*.go" -printf "%h\n" | sort -u | sed -n 's/\.\/\(.*\)/\1\/\1/p')"
```

Comment:
It seems like it is the wrong fix ^
Shouldn't we add an "ignore" for the linter check? 
@iaguis @alepuccetti @schu